.env.example

```diff
@@ -0,0 +1,2 @@
+token=yourgithubtoken
+DATABASE_URL=yourpostgresurl
```

Comment:
What's the formatting for this URL?
I tried running it with the value `localhost` and `localhost:5432`but it kept trying to connect to a database with that name, always on `localhost`.
Could you make this field's use case clearer?

Reply:
For sure,
Sorry for not making that clear :)

postgres://YourUser:YourPassword@YourServer:YourPort